gravitee-apim-console-webui/src/management/settings/portal-ng/portal-ng.component.ts

```diff
@@ -0,0 +1,473 @@
+          }),
+        }),
+        customApiKey: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.plan.security.customApiKey.enabled,
+            disabled:
+              this.isReadonly('plan.security.apikey.allowCustom.enabled') ||
+              settingsPermission ||
+              !this.settings.plan.security.apikey.enabled,
+          }),
+        }),
+        sharedApiKey: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.plan.security.sharedApiKey.enabled,
+            disabled:
+              this.isReadonly('plan.security.apikey.sharedApiKey.enabled') ||
+              settingsPermission ||
+              !this.settings.plan.security.apikey.enabled,
+          }),
+        }),
+        oauth2: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.plan.security.oauth2.enabled,
+            disabled: this.isReadonly('plan.security.oauth2.enabled') || settingsPermission,
+          }),
+        }),
+        jwt: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.plan.security.jwt.enabled,
+            disabled: this.isReadonly('plan.security.jwt.enabled') || settingsPermission,
+          }),
+        }),
+        push: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.plan.security.push.enabled,
+            disabled: this.isReadonly('plan.security.push.enabled') || settingsPermission,
+          }),
+        }),
+      }),
+      api: new FormGroup({
+        labelsDictionary: new FormControl({
+          value: this.settings.api.labelsDictionary,
+          disabled: this.isReadonly('api.labelsDictionary') || settingsPermission,
+        }),
+        primaryOwnerMode: new FormControl({
+          value: this.settings.api.primaryOwnerMode,
+          disabled: this.isReadonly('api.primaryOwnerMode') || settingsPermission,
+        }),
+      }),
+      dashboards: new FormGroup({
+        apiStatus: new FormGroup({
+          enabled: new FormControl({
+            value: this.settings.dashboards.apiStatus.enabled,
+            disabled: this.isReadonly('console.dashboards.apiStatus.enabled') || settingsPermission,
+          }),
+        }),
+      }),
+      portal: new FormGroup({
+        apikeyHeader: new FormControl({
+          value: this.settings.portal.apikeyHeader,
+          disabled: this.isReadonly('portal.apikey.header') || settingsPermission,
+        }),
+        url: new FormControl({
+          value: this.settings.portal.url,
+          disabled: this.isReadonly('portal.url') || settingsPermission,
+        }),
+        homepageTitle: new FormControl({
+          value: this.settings.portal.homepageTitle,
+          disabled: this.isReadonly('portal.homepageTitle') || settingsPermission,
+        }),
+        homepageTitleToggle: new FormControl(this.settings.portal.homepageTitle),
```

Comment:
Is it normal that the homepageTitleToggle is not disabled if missing the settingsPermission?

Reply:
Done